eng/configurations/targetgroups.props

```diff
@@ -6,8 +6,6 @@
       <TargetFrameworkIdentifier>.NETCore</TargetFrameworkIdentifier>
       <TargetFrameworkVersion>v5.0</TargetFrameworkVersion>
       <TargetingPackNugetPackageId>Microsoft.TargetingPack.Private.NetNative</TargetingPackNugetPackageId>
-      <NuGetTargetMoniker>.NETCore,Version=v5.0</NuGetTargetMoniker>
```

Comment:
Do these all come from the nuget in the sdk now?

Reply:
They were only required by the old target, in new target it uses TargetFramework and RuntimeIdentifier.